imp.py

```
@@ -0,0 +1,113 @@
+"""Compatibility shim for the removed :mod:`imp` module on Python 3.12+.
```

Ruff (0.15.14)

[error] Module `imp` shadows a Python standard-library module

(A005)